tools/upgrade.sh

```diff
@@ -1,13 +1,23 @@
-# Use colors, but only if connected to a terminal, and that terminal
-# supports them.
+#!/usr/bin/env zsh
+
+cd "$ZSH"
+
+# Use colors, but only if connected to a terminal
+# and that terminal supports them.
+
+local -a RAINBOW
+local RED GREEN YELLOW BLUE UNDER BOLD RESET
```

Comment:
Those 2 lines are wrong:
```
/home/mateusz/.oh-my-zsh/tools/upgrade.sh: 8: local: not in a function
```

Reply:
How are you running the file? It should use zsh instead of `sh`.

Reply:
Sorry my bad, you are right.

Reply:
This was changed recently. Where you running it manually? If so you should use `omz update` if at all possible.

I also added a check now to make sure it's always run with zsh if it was called with `sh`.

Reply:
I was using Topgrade, the fix was committed recently: https://github.com/r-darwish/topgrade/commit/a68387abed301221b7fef05b800059af2e87d38e and published soon after my comment here.